pkg/cmd/container/kill.go

```diff
@@ -65,6 +66,9 @@ func Kill(ctx context.Context, client *containerd.Client, reqs []string, options
 }
 
 func killContainer(ctx context.Context, container containerd.Container, signal syscall.Signal) error {
+	if err := containerutil.UpdateExplicitlyStoppedLabel(ctx, container, true); err != nil {
+		return err
+	}
 	task, err := container.Task(ctx, cio.Load)
```

Comment:
is it possible to set also `nerdctl/error` label as we did for `nerdctl stop`  [see](https://github.com/containerd/nerdctl/blob/main/pkg/containerutil/containerutil.go#L320-L324)
Thanks

Reply:
Thanks @fahedouch for the PR review.
It has been changed :-)